src/containers/Tenant/Query/Issues/Issues.tsx

```diff
@@ -26,48 +29,108 @@ const blockIssue = cn('kv-issue');
 interface ResultIssuesProps {
     data: ErrorResponse | string;
     hideSeverity?: boolean;
+
+    detailsMode?: IssuesViewMode;
 }
 
-export function ResultIssues({data, hideSeverity}: ResultIssuesProps) {
-    const [showIssues, setShowIssues] = React.useState(false);
+export function ResultIssues({
+    data,
+    hideSeverity,
+    detailsMode = ISSUES_VIEW_MODE.INLINE,
```

Comment:
Personally, I would not add this property. Instead, I’d recommend splitting the component into two separate components. As it stands, the current implementation violates the single responsibility principle, which makes the code harder to read and maintain.